test/Rings-conformance-tests.jl

```diff
@@ -99,6 +116,7 @@ end
 
 
 # helper
+equality(a, b) = isequal(a, b)
```

Comment:
I want to point out that for other exact types we use `==` instead of `isequal` (three lines below). Is there a reason for this discrepancy?

Reply:
No, no good reason, this is just not yet done and I wanted to hack something together quickly. It needs more thought to deal with inexact types, for example.